### DISCUSSION TITLE
edit a cell

### DISCUSSION THREAD

**Discussion by drinkingwine** [shadcn-ui/ui#4369] [Q&A]
Kinda newbie here. Looking for a suggestion on how to make a cell in a shadcn data table editable inline?
Thanks in advance

**Comment by mithatercan** [comment]
You can follow the pattern shown in the [Tanstack Table example](https://tanstack.com/table/v8/docs/framework/react/examples/editable-data) for editable data. Basically add an updateData function in the meta prop to handle updates from the table cells.
